lunavl/sdk/estimators/face_estimators/head_pose.py

```diff
@@ -173,3 +174,28 @@ def estimateByBoundingBox(self, detection: BoundingBox, imageWithDetection: VLIm
         if error.isError:
             raise LunaSDKException(LunaVLError.fromSDKError(error))
         return HeadPose(headPoseEstimation)
+
+    @CoreExceptionWrap(LunaVLError.EstimationHeadPoseError)
+    def estimateByBoundingBoxBatch(
+        self, detections: List[BoundingBox], imageWithDetectionList: List[VLImage]
```

Comment:
There is detection structure, there is other structures

Reply:
some structures added